src/components/MainNavigation/README.mdx

```diff
@@ -8,24 +8,13 @@ import { List, ListItem } from '../List';
 
 #### When to use
 
-<List kind="bullet">
-  <ListItem kind="checkmark">Internal and external applications</ListItem>
-  <ListItem kind="checkmark">
-    The MainNavigation can be used for both internal and external websites.
-  </ListItem>
-</List>
+- ✓ Link to emails or phone numbers
```

Comment:
Please explain what is `Link to emails or phone numbers`?

Reply:
Will correct that, an error on my end